src/components/floatingButton/index.tsx

```diff
@@ -199,7 +194,8 @@ class FloatingButton extends PureComponent<FloatingButtonProps> {
     return (
       <View
         row={!!this.isSecondaryHorizontal}
-        center={!!this.isSecondaryHorizontal}
+        center={!fullWidth}
+        paddingH-16={fullWidth}
```

Comment:
Why use the `padding` here?
Is it by design?

In the example screen set ti Horizontal layout and play with the `fullWidth` you can see it changes even though it shouldn't.

Reply:
I see. Thanks. Fixed